controllers/error-handling-controller.js

```diff
@@ -0,0 +1,25 @@
+const app = require('../app');
+
+//will need conditional to be expanded as cases are found, have a feeling i need to implement something further 
+//in models? not sure...
+const handle404NotFound = (err, req, res, next) => {
```

Comment:
I would recommend moving the 404 error handling into app.js. The 500 can stay inside this file however. 
You may want to look at how you can check to see if a path is invalid and use the method to send your 404 message in app.js